Firmware/Main/PreCharge.cpp

```diff
@@ -116,7 +120,7 @@ void preChargeCircuitFSMTransitions (PreChargeTaskData preChargeData) {
         // kill-switch activated or HV switch turned off
         hv_state = HV_OFF;
       }
-      else if (!isHVSafe(preChargeData) || gyro_kalman->angle_Y > 45 || gyro_kalman->angle_Y < -45 || gyro_kalman->angle_X > 45 || gyro_kalman->angle_X < -45) {
+      else if (!isHVSafe(preChargeData) || !isAngleSafe(preChargeData)) {
```

Comment:
Need to do a lot of testing with the gyro on the bike to make sure it has no bugs. It would be really bad if the contactor shuts off randomly without the rider expecting it.

Reply:
ok! Do you think I should just comment out the gyro stuff for now then?

Also ik there's a comment about the gyroscope outputting NaN a lot, but I've never actually seen that happen while I've been testing?